README.md

```diff
@@ -87,6 +87,109 @@ with Maven.
 This project uses the [Google java code style](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html)
 to keep code clean and consistant.
 
+Maven Plugin
+-----------
+
+## schema-registry:download
+
+This plugin is used to download AVRO schemas for the requested subjects.
+
+|        Name        |                                                      Description                                                      |  Type  | Required | Default |
+|:------------------:|:---------------------------------------------------------------------------------------------------------------------:|:------:|:--------:|:-------:|
+| schemaRegistryUrls | Urls for the schema registry instance to connect to                                                                   |  List  |    yes   |         |
+| outputDirectory    | Output directory to write the schemas to.                                                                             | String |    yes   |         |
+| schemaExtension    | The file extension to use for the output file name. This must begin with a '.' character.                             | String |    no    |  .avsc  |
+| subjectPatterns    | The subject patterns to download. This is a list of regular expressions. Patterns must match the entire subject name. | List   |    no    |   ^.+$  |
+
+```
+<plugin>
+    <groupId>io.confluent</groupId>
+    <artifactId>kafka-schema-registry-maven-plugin</artifactId>
+    <version>3.1.0-SNAPSHOT</version>
+    <configuration>
+        <schemaRegistryUrls>
+            <param>http://192.168.99.100:8081</param>
```

Comment:
Will it be possible to use profiles or something to customize this by dev/staging/prod?